### DISCUSSION TITLE
Build not running locally to give node - some number

### DISCUSSION THREAD

**Discussion by deepEngineering23** [official-stockfish/Stockfish#5060] [General]
make build
./stockfish
Stockfish dev-20240217-fc41f64d by the Stockfish developers (see AUTHORS file) stuck on this

**Comment by PGG106** [comment]
assuming you want a bench nodecount you need to send a bench command 
`./stockfish bench`
currently you are just opening the exe and sending nothing, it's not stuck, it's waiting for your input

**Reply by deepEngineering23** [reply]
oh forgot it ! my fault
